lib/Jsrt/ChakraCore.h

```diff
@@ -163,7 +163,7 @@ JsGetModuleHostInfo(
     _In_ JsModuleHostInfoKind moduleHostInfo,
     _Outptr_result_maybenull_ void** hostInfo);
 
-#ifndef NTBUILD
+#ifdef CHAKRACOREBUILD_
```

Comment:
Could you elaborate why we need this `#ifdef` (either NTBUILD or CHAKRACOREBUILD_)? My understanding is that `ChakraCore.h` is the header for ChakraCore only thus is always CHAKRACOREBUILD. NTBUILD does not use this header at all.